_assets/css/custom/_utilities.scss

```diff
@@ -22,3 +22,17 @@
 .grid-gap-xl {
   @include grid-gap(6);
 }
+
+// Custom margin additions
```

Comment:
I think this is perfectly fine, so feel free to take or leave this suggestion, but another way to accomplish this would be to add these values to `_uswds-theme-utilities.scss` like:

```
$margin-manual-values: (12: 6.5rem, 16: 8rem, 20: 10rem);
```
Main benefit with this approach would be for consistency, making those new steps available for all variations of margin.